### ISSUE TITLE
POM file needs fixing to support Sonatype deployments

### ISSUE THREAD

**Issue by ibaldin** [RENCI-NRIG/xmpp-pubsub#4]
This needs to be addresses
```
[ERROR]   Rule "sources-staging" failures
[ERROR]     * Missing: no sources jar found in folder '/org/renci/xmpp-pubsub/1.1'
[ERROR]   Rule "signature-staging" failures
[ERROR]     * Missing Signature: '/org/renci/xmpp-pubsub/1.1/xmpp-pubsub-1.1-javadoc.jar.asc' does not exist for 'xmpp-pubsub-1.1-javadoc.jar'.
[ERROR]     * Missing Signature: '/org/renci/xmpp-pubsub/1.1/xmpp-pubsub-1.1.pom.asc' does not exist for 'xmpp-pubsub-1.1.pom'.
[ERROR]     * Missing Signature: '/org/renci/xmpp-pubsub/1.1/xmpp-pubsub-1.1.jar.asc' does not exist for 'xmpp-pubsub-1.1.jar'.
[ERROR] 
```

**Comment by ibaldin** [comment]
Now published in https://oss.sonatype.org/#nexus-search;quick~xmpp-pubsub

**Comment by ibaldin** [comment]
Fixed in #3
